public/app/core/components/Select/UserPicker.tsx

```diff
@@ -42,7 +42,8 @@ export class UserPicker extends Component<Props, State> {
       .then((result: OrgUser[]) => {
         return result.map((user) => ({
           id: user.userId,
-          value: user.userId,
+          uid: user.userId,
```

Comment:
Should this be:
```suggestion
          uid: user.uid,
```

Reply:
:sweat_smile: missed that one, fortunately it's not used